Library/Homebrew/standalone/init.rb

```diff
@@ -29,7 +29,7 @@
 require "pathname"
 dir = __dir__ || raise("__dir__ is not defined")
 HOMEBREW_LIBRARY_PATH = Pathname(dir).parent.realpath.freeze
-HOMEBREW_PORTABLE_RUBY = Pathname.new(RbConfig.ruby).to_s.include?("/vendor/portable-ruby/").freeze
+HOMEBREW_USING_PORTABLE_RUBY = Pathname.new(RbConfig.ruby).to_s.include?("/vendor/portable-ruby/").freeze
```

Comment:
```suggestion
HOMEBREW_USING_PORTABLE_RUBY = RbConfig.ruby.include?("/vendor/portable-ruby/").freeze
```
I think this should just work?

Reply:
@carlocab Will push this to the next PR, incoming shortly 😁

Reply:
@carlocab done in https://github.com/Homebrew/brew/pull/17504